ush/narre/evs_get_prepbufr.sh

```diff
@@ -19,16 +19,16 @@ if [ $modnam = prepbufr ] ; then
 
  for vhr in 00  03  06  09  12  15  18   21  ; do
   if [ -s $COMINobsproc/rap.${VDATE}/rap.t${vhr}z.prepbufr.tm00 ] ; then 
+   split_by_subset $COMINobsproc/rap.${VDATE}/rap.t${vhr}z.prepbufr.tm00
+   cat $WORK/ADPSFC $WORK/SFCSHP $WORK/MSONET>> $WORK/prepbufr.$VDATE/rap.t${vhr}z.prepbufr.tm00
```

Comment:
Can we get some file checking for the outputs from `split_by_subset` before cat'ing? Maybe something like

`for subset in ADPSFC SFCSHP MSONET; do`
`    if [ -s ${WORK}/${subset} ]; then`
`        cat ${WORK}/${subset} >> $WORK/prepbufr.$VDATE/rap.t${vhr}z.prepbufr.tm00`
`    fi`
`done`